packages/experiments-realm/crm/contact.gts

```diff
@@ -0,0 +1,392 @@
+  };
+}
+
+class FittedTemplate extends Component<typeof Contact> {
+  private formatPhone(phoneMobile: any) {
+    if (!phoneMobile) return undefined;
+    return `+${phoneMobile.country} ${phoneMobile.area}-${phoneMobile.number}`;
+  }
+
+  private formatEmail(email: string) {
+    if (!email) return undefined;
+    return email;
+  }
+
+  private openSocialLink(url: string) {
+    window.open(url, '_blank');
+  }
+
+  private getStatusIcon(label: string | undefined) {
+    switch (label) {
+      case 'Customer':
+        return {
+          icon: HeartHandshake,
+          lightColor: '#8bff98',
+          darkColor: '#01d818',
+        };
+      case 'Lead':
+        return {
+          icon: TargetArrow,
+          lightColor: '#E6F4FF',
+          darkColor: '#0090FF',
+        };
+      default:
+        return null;
+    }
+  }
+
+  <template>
+    <article class='fitted-contact-card'>
+      <div class='avatar-container'>
+        <div class='avatar'>
+          <img src={{@model.thumbnailURL}} alt={{@model.name}} />
+        </div>
+        <div class='avatar-info'>
+          <h3 class='name'>{{if @model.name @model.name 'Unnamed Contact'}}</h3>
+          <div class='row'>
+            <Building class='icon' />
+            <span>{{@model.company}}</span>
+          </div>
+        </div>
+      </div>
+
+      <div class='contact-info'>
+        {{#if @model.primaryEmail}}
+          <div class='row'>
+            <Mail class='icon gray' />
+            <span>{{this.formatEmail @model.primaryEmail}}</span>
+          </div>
+        {{/if}}
+
+        {{#if @model.secondaryEmail}}
+          <div class='row'>
+            <Mail class='icon gray' />
+            <span>{{this.formatEmail @model.secondaryEmail}}</span>
+          </div>
+        {{/if}}
+
+        {{#if @model.phoneMobile}}
+          <div class='row'>
+            <Phone class='icon gray' />
+            <span>{{this.formatPhone @model.phoneMobile}}</span>
+            <Pill class='pill-gray'>
+              mobile
+            </Pill>
+          </div>
+        {{/if}}
+
+        {{#if @model.phoneOffice}}
+          <div class='row'>
+            <Phone class='icon gray' />
+            <span>{{this.formatPhone @model.phoneOffice}}</span>
+            <Pill class='pill-gray'>
+              office
+            </Pill>
+          </div>
+        {{/if}}
+      </div>
+
+      <div class='social-links'>
+        {{#if @model.twitterURL}}
+          <IconButton
+            @onClick={{fn this.openSocialLink @model.twitterURL}}
+            @icon={{BrandTwitter}}
+            @width='20'
+            @height='20'
+            class='social-link'
+          />
+        {{/if}}
+        {{#if @model.linkedInURL}}
+          <IconButton
+            @onClick={{fn this.openSocialLink @model.linkedInURL}}
+            @icon={{BrandLinkedin}}
+            @width='20'
+            @height='20'
+            class='social-link'
+          />
+        {{/if}}
+      </div>
```

Comment:
I think it makes sense for the social media links to be a compound field and have it render a list of icons